frontend/src/components/GoalsTable/GoalsTable.js

```diff
@@ -0,0 +1,204 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Table, Grid, Alert,
+} from '@trussworks/react-uswds';
+import { filtersToQueryString } from '../../utils';
+import TableHeader from '../TableHeader';
+import Container from '../Container';
+import GoalRow from './GoalRow';
+import { GOALS_PER_PAGE } from '../../Constants';
+import './GoalTable.css';
+import { getRecipientGoals } from '../../fetchers/recipient';
+
+function GoalsTable({
+  recipientId,
+  filters,
+  onUpdateFilters,
+}) {
+  // Goal Data.
+  const [goals, setGoals] = useState([]);
+
+  // Page Behavior.
+  const [error, setError] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  // Grid and Paging.
+  const [activePage, setActivePage] = useState(1);
+  const [goalsCount, setGoalsCount] = useState(0);
+  const [offset, setOffset] = useState(0);
+  const [perPage] = useState(GOALS_PER_PAGE);
+  const [sortConfig, setSortConfig] = React.useState({
+    sortBy: 'goalStatus',
```

Comment:
how come React.useState instead of just useState?

Reply:
Updated to regular useState().